database/connection.py

```diff
@@ -1,15 +1,23 @@
 import pymysql
+import pymysql.cursors
+import cx_Oracle
 
 #database connection
-connection = pymysql.connect(host="localhost", user="root", passwd="", database="test")
-cursor = connection.cursor()
-select_tbl= "select * from tbl_employee "
-cursor.execute(select_tbl)
+def db_connect():
+
+    db = pymysql.connect(host="127.0.0.1", port=3306,user="root",password="",database="test")
+    #db =cx_Oracle.connect('scott/tiger@localhost')
+    if (db):
+        print("Connection established:",db)
+        print(db.host)
```

Comment:
Remove this print statement. The previous print will host details as well.

Reply:
Removed print (db.host) line